src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -179,6 +181,7 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		renderType.setFont(renderType.getFont().deriveFont(16.0f));
 		renderType.setPreferredSize(new Dimension(350, 42));
 		renderType.setRenderer(new ModelComboBoxRenderer());
+		renderType.addActionListener(e -> updateFields());
```

Comment:
Is this still needed?

Reply:
Sorry, hard to keep up with everything on a project not created by me.

Reply:
> Just remembered your previous input on blocking model restrictions 🤷‍♂️

So it is needed?

Reply:
Given we canceled our restrictions decision, no.